submit-web/src/components/Submission/DocumentRow.tsx

```diff
@@ -4,19 +4,39 @@ import { useState } from "react";
 import { notify } from "../Shared/Snackbar/snackbarStore";
 import { getObjectFromS3 } from "../Shared/Table/utils";
 import { StyledTableCell } from "../Shared/Table/common";
+import { openModal } from "../Shared/Modals/modalStore";
+import ConfirmationModal from "../Shared/Modals/ConfirmationModal";
+import { SubmissionItemTableRow } from "./types";
 
 type DocumentRowProps = {
   documentSubmission: Submission;
+  submissionItem: SubmissionItemTableRow;
 };
 
-export default function DocumentRow({ documentSubmission }: DocumentRowProps) {
+export default function DocumentRow({
+  documentSubmission,
+  submissionItem,
+}: DocumentRowProps) {
   const [pendingGetObject, setPendingGetObject] = useState(false);
   const {
     submitted_document: { name, url },
     version,
     submitted_by,
   } = documentSubmission;
 
+  const openVerificationModal = () => {
+    //todo: setup onConfirm
+    openModal(
+      <ConfirmationModal
+        onConfirm={() => {}}
```

Comment:
okay, so what happens when they click confirm?

They should proceed to open the document right?

Reply:
update status then open yes